source/features/download-folder-button.tsx

```diff
@@ -39,7 +42,7 @@ void features.add({
 		pageDetect.isRepoTree
 	],
 	exclude: [
-		pageDetect.isRepoRoot // Already has an native download ZIP button
+		() => select.exists('get-repo-controller') // Already has an native download ZIP button
```

Comment:
I think this specific change can be dropped if you use the [v2](https://github.com/fregante/github-url-detection/releases/tag/v2.0.0) thanks to https://github.com/fregante/github-url-detection/pull/16

Reply:
I've reverted the commit; the package will be updated in #3308.